factory/read-config.js

```diff
@@ -6,7 +6,15 @@ const __dirname = path.dirname(fileURLToPath(import.meta.url));
 
 export default function (configPath = './config.json') {
   const fullConfigPath = path.resolve(__dirname, '../', configPath);
-  if (!fs.existsSync(fullConfigPath)) throw new Error('Config file does not exist.');
+  const fullExamplePath = path.resolve(__dirname, '../', './example-config.json');
+
+  if (!fs.existsSync(fullConfigPath)) {
+    try{
+      fs.copyFileSync(fullExamplePath, fullConfigPath);
+      throw new Error("SquadJS couldn't find any configuration file. We have generated a config file at "+fullConfigPath+". This file needs to be configured before running JquadJS again.");
+    } catch (err) { throw err; }
```

Comment:
Can get rid of the try/catch here as you're just rethrowing the error.

Reply:
the try catch is just for the case where the original file is missing, but is suppose they have bigger problems if that's the case

Reply:
The `if (!fs.existsSync(fullConfigPath)) {` block catches the cases if the file does not exist. There's no point in the try catch block as it's catching the error and just throwing it again.